projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -1251,12 +1328,12 @@ public function get_related_post_data_for_post( $post_id, $position, $origin ) {
 			 * @param string $this->_to_utf8( $this->_generate_related_post_context( $post->ID ) ) Context displayed below each related post.
```

Comment:
May as well fix the function refs in the comment too.
```suggestion
			 * @param string $this->to_utf8( $this->generate_related_post_context( $post->ID ) ) Context displayed below each related post.
```

Reply:
Nice catch!